scripts/token-mold.js

```diff
@@ -1285,30 +1391,37 @@ class TokenMoldForm extends FormApplication {
     };
 
     let barAttributes = [];
-    for (let type in game.system.model.Actor) {
-      let newAttributes = getAttributes(game.system.model.Actor[type]).map(
-        (e) => e.join(".")
+
+    const types = Actor.implementation.TYPES;
+    const shellMap = new Map(
+      types.map((t) => [t, new Actor.implementation({ name: t, type: t })]),
+    );
+    shellMap.forEach((value, key, map) => {
+      const newAttributes = getAttributes(value.toObject().system).map((e) =>
+        e.join("."),
```

Comment:
New `map` code.

Reply:
I'm still not sure you need the toObject here:  the props of `npc.system` stilll contain what you want to pull out, just more of it.  However, the toObject() call strips off all the primitives and additional functions and model data from CreatureTemplate, which makes the object leaner.  Effectively, you're extracting all of the iterable props from the npc.system object to re-configure just the parts needed.  no objection;  there's not enough of a performance hit where we care.